Makefile

```diff
@@ -71,7 +72,7 @@ build-full:
 
 	cd nxcompshad && autoconf && (${CONFIGURE}) && ${MAKE}
 
-	cd nx-X11 && ${MAKE} World
+	cd nx-X11 && ${MAKE} World USRLIBDIR=$(USRLIBDIR) SHLIBDIR=$(SHLIBDIR)
```

Comment:
We only have libNX_X11 left. And libNX_X11 is intalled into SHLIBDIR by our main Makefile. We don't use InstallLibrary() for nx-libs.

This will go away with the autoconf/automake migration, so let's bother about this here. If needed, I could remove the InstallLibrary() code from the imake stuff.